### ISSUE TITLE
Raw store is not available anymore

### ISSUE THREAD

**Issue by edilsalvador** [Spyna/react-context-hook#14]
Hi,

I've been using this library to my project since version 1.
Lately I upgraded from v1.2.1 to v2.1.1 and I read that the store is now using redux.

Some parts of my code uses the exported rawStore to update the state of my store without the need to subscribe to any hooks. Here's a sample code:

```
export const resetStore = () => {
  Object.keys(initialState).map(key => {
    store.set([key], initialState[key]);
  });
};
```
Now that the rawStore is gone, is there a way to update the store without using hooks?

Thanks!

**Comment by Spyna** [comment]
Hi, sorry for the inconvenience. 
I'll restore the "rawStore" with a "reset" function.

**Comment by Spyna** [comment]
Issue resolved in version 2.2.0. 
More info on how to use it: https://github.com/Spyna/react-context-hook#using-store-outside-of-react-components
